docs/development/code-quality.md

```diff
@@ -0,0 +1,27 @@
+# Linting and formatting
+
+The whole project can be linted explicitly at once (python and javascript) by running `make lint`.
+
+Run `./bin/lint --check-deps-only` to check if all dependencies have been installed
+
+## The lint tool
+
+There is a python tool at ./bin/lint which proxies some of the linting tools. It takes care of
+routing to different actions based on if it's a CI build or not.
+
+## Javascript linting
+
+The javascript code is linted using `eslint`. Relevant configuration files are:
+
+Explicit command: `make lint-js`
+
+* .eslintignore: Files to be ignored during linting
+* .eslintrc.js
+
+## Javascript formatting
```

Comment:
Perhaps we should write explicitly what's included in linting? I for my self is not so sure what's included other than formatting and unit tests.

Reply:
Yes good point, I hope to get back to this when I have time :) For now, I hope that the image that I added (but forgot to link to in this document) gives enough information, because it says which configuration files are being used. I'll add a link to it now.

Reply:
good enough!